src/Raven.Client/Documents/Commands/UpdateSubscriptionCommand.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Net.Http;
+using Raven.Client.Documents.Session;
+using Raven.Client.Documents.Subscriptions;
+using Raven.Client.Http;
+using Raven.Client.Json;
+using Raven.Client.Json.Converters;
+using Raven.Client.Util;
+using Sparrow.Json;
+
+namespace Raven.Client.Documents.Commands
+{
+    internal class UpdateSubscriptionCommand : RavenCommand<CreateSubscriptionResult>, IRaftCommand
```

Comment:
CreateSubscriptionResult -> UpdateSubscriptionResult

Reply:
CreateSubscriptionResult just have a subscription name
https://github.com/ravendb/ravendb/blob/v4.2/src/Raven.Client/Documents/Subscriptions/CreateSubscriptionResult.cs#L5

Reply:
Just inherit from it, that is all.